docs/tutorials/installation.md

```diff
@@ -21,31 +20,34 @@ $ pip install "jax[cuda11_pip]" -f https://storage.googleapis.com/jax-releases/j
 # Follow Apple's instructions at https://developer.apple.com/metal/jax/
 ```
```

Comment:
I think we can remove this block, because it duplicates what is in the table just below this.